### PR TITLE
Replace script templates with a worker module

### PR DESCRIPTION
**Pull request by cdonati**
The removes the dynamic script generation that the taskqueue was performing to create the celery worker scripts.

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2793/

**Comment by menivaitsi** [comment]
Is this going to work off the shelf for upgrades? Would it require an upgrade path?

**Comment by cdonati** [comment]
Yes, there is no upgrade required other than upgrading the taskqueue package.

File: AppTaskQueue/appscale/taskqueue/push_worker.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
+import datetime
```
Comment (menivaitsi) [inline AppTaskQueue/appscale/taskqueue/push_worker.py]: Add docstring to this file.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2865/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/3150/
The hawkeye failure was due to a connection timeout.

**Pull request merged**